File: c++/08_pgm_invoice_engine/src/co/edu/campusucc/poo/interfaces/IClient.cpp
#ifndef ICLIENT_H
#define ICLIENT_H

class IClient {
public:
    virtual void nombre() = 0;
    virtual void apellido() = 0;
    virtual void tipoTarjeta() = 0;
    virtual void numTarjeta() = 0;
};

#endif

File: python/java_to_python/08-pgmInvoiceEngine/src/co/edu/campusucc/poo/interfaces/IClient.py
from abc import ABC, abstractmethod

class IClient(ABC):
    @abstractmethod
    def nombre(self):
        pass
        
    @abstractmethod
    def apellido(self):
        pass
        
    @abstractmethod
    def tipoTarjeta(self):
        pass
        
    @abstractmethod
    def numTarjeta(self):
        pass